backend/apps/common/tests.py

```
@@ -425,3 +429,199 @@ def test_student_profile_not_found(self):
+        domain.domain = cls.get_test_tenant_domain()
+        domain.is_primary = True
+
+    def setUp(self):
+        super().setUp()
+        user_model = get_user_model()
+
+        # Usuarios
+        self.admin_user = user_model.objects.create_user(
+            username="admin-nexus",
+            email="admin@nexus.test",
+            password="PASSWORD1",
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to argument: "password"

(S106)